wire-ios-data-model/Source/Utilis/EAR/EARKeyRepository.swift

```diff
@@ -34,6 +35,7 @@ protocol EARKeyRepositoryInterface {
 
 }
 
+/// Caches keys for reuse and avoid prompting the user to authenticate for each key access.
```

Comment:
Good point! I have played with the sample project from Apple and unfortunately after a few minutes even a cached `LAContext` prompts again. So it would be counteracting to our wanted behavior if we would remove the local key cache.
https://developer.apple.com/documentation/localauthentication/accessing-keychain-items-with-face-id-or-touch-id